poms/build-parent/pom.xml

```diff
@@ -97,6 +97,7 @@
                     <version>${formatter-maven-plugin.version}</version>
                     <configuration>
                         <configFile>${maven.multiModuleProjectDirectory}/tooling/eclipse-formatter-config.xml</configFile>
+                        <lineEnding>KEEP</lineEnding>
```

Comment:
What's the meaning of this change? Would the plugin keep Windows EOLs if end users enter them?

Reply:
Oh really? Keep is certainly better than user's platform. Is there no way to force Unix EOLs?

Reply:
>  Is there no way to force Unix EOLs?

Yes, I guess it'd make more sense to use `<lineEnding>LF</lindending>`.

Reply:
Latest commit forces Unix EOLs.